package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "connect-mongo": "^1.3.2",
     "connect-redis": "^3.1.0",
+    "crypto-js": "^3.1.9-1",
```

Comment:
This is a pre-release version of crypto-js . It should use a release version (e.g. `3.1.9`).